pkg/services/apiserver/standalone/factory.go

```diff
@@ -33,6 +34,9 @@ type APIServerFactory interface {
 	// Given the flags, what can we produce
 	GetEnabled(runtime []RuntimeConfig) ([]schema.GroupVersion, error)
 
+	// Any optional middlewares this factory wants configured via apiserver's BuildHandlerChain facility
+	GetOptionalMiddlewares(tracer tracing.Tracer) []func(handler http.Handler) http.Handler
```

Comment:
This is one way I found how to specify such middlewares because standalone mode doesn't really have the web/router setup which executes `contexthandler.Middleware`. Open to other suggestions.